pkg/bucketeer/model/metrics_event.go

```diff
@@ -31,8 +31,15 @@ type metricsDetailEventType string
 
 //nolint:lll
 const (
-	GetEvaluationLatencyMetricsEventType metricsDetailEventType = "type.googleapis.com/bucketeer.event.client.GetEvaluationLatencyMetricsEvent"
-	GetEvaluationSizeMetricsEventType    metricsDetailEventType = "type.googleapis.com/bucketeer.event.client.GetEvaluationSizeMetricsEvent"
-	TimeoutErrorCountMetricsEventType    metricsDetailEventType = "type.googleapis.com/bucketeer.event.client.TimeoutErrorCountMetricsEvent"
-	InternalErrorCountMetricsEventType   metricsDetailEventType = "type.googleapis.com/bucketeer.event.client.InternalErrorCountMetricsEvent"
+	LatencyMetricsEventType          metricsDetailEventType = "type.googleapis.com/bucketeer.event.client.LatencyMetricsEvent"
+	SizeMetricsEventType             metricsDetailEventType = "type.googleapis.com/bucketeer.event.client.SizeMetricsEvent"
+	TimeoutErrorMetricsEventType     metricsDetailEventType = "type.googleapis.com/bucketeer.event.client.TimeoutErrorMetricsEvent"
+	InternalSDKErrorMetricsEventType metricsDetailEventType = "type.googleapis.com/bucketeer.event.client.InternalSDKErrorMetricsEvent"
+)
+
+type APIID int
+
+const (
+	GetEvaluation  APIID = 1
+	RegisterEvents APIID = 3
```

Comment:
It seems that `RegisterEvents` is not used, is that OK?

Reply:
Talked at meeting.